### DISCUSSION TITLE
usbdm_rs_3.3.3-beta

### DISCUSSION THREAD

**Discussion by Kuraga13** [Kuraga13/usbdm_mc56f_rs#68] [Announcements]
Functionality as of 03/11/23 - full-fledged work (reading, writing, erasing, verification).
DSC | tested:
MC56F8002
MC56F8006 ✔️
MC56F8011
MC56F8013
MC56F8025 ✔️
MC56F8035 ✔️
There are two pre-builded exe's for Windows10-11 32-64 bit, in the archive at once, this is due to the peculiarities of the Windows graphics drivers and Iced renderer, somewhere OpenGL (iced-glow) works better, somewhere DirectX (iced-wgpu). In the future, we will try to either achieve stable operation from this engine, or switch to the classic brutal version a la QT.

The connection_image folder appeared, I did a fairly simple addition of a picture, that's enough for the first time. The name of the controller in capital letters and .jpeg format & extension.

Otherwise, the requirements is Windows10-11, usbdm full dsc, it is advisable to update the firmware immediately

<hr /><em>This discussion was created from the release <a href='https://github.com/Kuraga13/usbdm_mc56f_rs/releases/tag/'>usbdm_rs_3.3.3-beta</a>.</em>

**Comment by specialmanvn01** [comment]
hello, thank you for your post. I'm trying to add device MC56F82746 to your project. but i cannot get the target speed, because i'm using file base_routine for 80xx then i got 00. Can you post your code for base routine. Thank you

**Comment by Kuraga13** [comment]
Hi, 
you can use this one, but need some additional work around FTFA registers on init.
Also you can use our soft — MultiProg, full version have this controllers 
https://kuraga.tech/docs/MultiProg/overview
>Вторник, 21 октября 2025, 18:22 +03:00 от specialmanvn01 < ***@***.*** >:
> 
> 
>hello, thank you for your post. I'm trying to add device MC56F82746 to your project. but i cannot get the target speed, because i'm using file base_routine for 80xx then i got 00. Can you post your code for base routine. Thank you
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you authored the thread. Message ID:  <Kuraga13/usbdm_mc56f_rs/repo-discussions/68/comments/14741773 @ github . com>
--
Курамшин Тимур
+ 7 965 016 71 94
kuraga-remont.ru
